### ISSUE TITLE
There is no help command

### ISSUE THREAD

**Issue by Brycey92** [MinecraftPortCentral/GriefPrevention#777]
The wiki states `/gphelp` is the help command, but ingame with GP 4.3.0.653, the command doesn't exist. The old help command, `/griefprevention` also doesn't exist.

**Comment by ktole1999** [comment]
what about /gp  the wiki may not be up to date with latest changes.

**Comment by Brycey92** [comment]
`/gp` doesn't exist either, I just checked.

**Comment by ktole1999** [comment]
Did u confirm the plugin loaded?



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by DrZoddiak** [comment]
https://gyazo.com/d488d10c739a8979caade11408c4044f

As you can see it exists, but is not implemented

**Comment by ktole1999** [comment]
Not sure what that link is but its not GP



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by DrZoddiak** [comment]
Sorry. Fixed that. I forgot to copy the link 🤦‍♂️

**Comment by ktole1999** [comment]
@Brycey92 not sure what you are trying to accomplish but entering a partial command and then hitting TAB should give available commands for what was entered.

**Comment by DrZoddiak** [comment]
You could implement the help command pretty easily. If you PR it, it might get added as well.

**Comment by Brycey92** [comment]
Good point. I assume I shouldn't just hardcode everything, right? Is there somewhere that command aliases, usages, and descriptions are stored that can be accessed programmatically?

**Comment by DrZoddiak** [comment]
I'm not familiar with GP codebase. You'll have to take a gander at it yourself, or ask Blood possibly.

**Comment by bloodmc** [comment]
Next major GP update will include a new `/gp` command to handle everything along with help.
